components/tools/OmeroWeb/omeroweb/webclient/urls.py

```diff
@@ -35,9 +35,6 @@
     # Home page is the main 'Data' page
     url(r'^$', views.load_template, {'menu': 'userdata'}, name="webindex"),
 
-    # 'Feed' / 'recent'
-    url(r'^feed/$', views.feed, name="web_feed"),
```

Comment:
any objections to remove these views? I doubt anyone uses them. @chris-allan @will-moore